src/mcp-server.ts

```
@@ -111,173 +140,248 @@ export class AzureDevOpsMCPServer {
           type: 'text',
-          text: JSON.stringify(items, null, 2)
-        }
-      ]
-    };
+          text: JSON.stringify(items, null, 2),
+        },
+      ],
+    }
   }
 
   private async handleQueryWork(args: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)